core/user/errors.go

```diff
@@ -3,10 +3,13 @@ package user
 import "errors"
 
 var (
-	ErrNotExist     = errors.New("user doesn't exist")
-	ErrInvalidID    = errors.New("user id is invalid")
-	ErrInvalidEmail = errors.New("user email is invalid")
-	ErrMissingEmail = errors.New("user email is missing")
-	ErrInvalidUUID  = errors.New("invalid syntax of uuid")
-	ErrConflict     = errors.New("user already exist")
+	ErrNotExist         = errors.New("user doesn't exist")
+	ErrInvalidID        = errors.New("user id is invalid")
+	ErrInvalidEmail     = errors.New("user email is invalid")
+	ErrNotUUID          = errors.New("invalid syntax of uuid")
```

Comment:
Looks like this is not being used anywhere. Do we need this?

Reply:
Yes, `ErrNotUUID ` isn't being used.